components/esm/creaturestats.hpp

```diff
@@ -36,7 +36,7 @@ namespace ESM
         std::vector<int> mSummonGraveyard;
 
         ESM::TimeStamp mTradeTime;
-        int mGoldPool;
+        unsigned int mGoldPool;
```

Comment:
Unsure about this one. Are we actually using unsigned int across the board for gold amounts? If not then this chance has the potential to introduce subtle bugs and inconsistencies.

Reply:
Not yet, as noted above by @akortunov, I haven't gotten to apps/openmw yet.